package.json

```diff
@@ -1,76 +1,75 @@
 {
   "name": "compose-teams",
-  "version": "0.0.0",
+  "version": "0.0.1",
+  "description": "Angular app to help you generate optimized teams for a sport (ex football).",
+  "keywords": [
+    "angular",
+    "angular 8",
+    "ngx-webstorage",
+    "rxjs",
+    "material"
+  ],
+  "repository": "https://github.com/maxime1992/compose-teams",
   "license": "MIT",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "test": "ng test",
     "lint": "ng lint",
+    "pree2e": "webdriver-manager update --standalone false --gecko false",
     "e2e": "ng e2e",
-    "check": "yarn run lint:check && yarn run prettier:check",
-    "check:fix": "yarn run lint:fix && yarn run prettier:fix",
-    "lint:check": "ng lint --type-check",
-    "lint:fix": "yarn run lint:check -- --fix",
-    "build:prod": "ng build --prod --build-optimizer --no-sourcemap",
-    "prettier:base": "yarn run prettier -- --single-quote --trailing-comma es5",
-    "prettier:base-files": "yarn run prettier:base -- './{e2e,src}/**/*.{scss,ts}'",
-    "prettier:fix": "yarn run prettier:base-files -- --write",
-    "prettier:check": "yarn run prettier:base-files -- -l",
-    "precommit": "lint-staged",
-    "prepush": "yarn run lint:check"
+    "dist": "ng build",
+    "prod": "ng build --prod",
+    "prod:src": "ng build --prod --sourcemaps"
   },
```

Comment:
@maxime1992  Updated but as I say, `team.component.scss` and `need-10-players.component.ts` seems cause problem with `Prettier`. And I can't push without validation.
I comment this checking at the moment.

Reply:
Do you mean that you haven't been able to push those files or that it will fail on CI?
Push them using --no-verify (from memory) and I'll take a look

Reply:
I pushed with --no-verify.

Reply:
@maxime1992 Did you take a look for prettier?

Reply:
Not really and to be honest I don't have much time for this project.
I think you should just work on your own fork =) !